Polyfills/Canvas/Source/Image.cpp

```diff
@@ -94,26 +94,29 @@ namespace Babylon::Polyfills::Internal
         request.SendAsync().then(m_runtimeScheduler, *m_cancellationSource, [env{info.Env()}, this, request{std::move(request)}](arcana::expected<void, std::exception_ptr> result) {
             if (result.has_error())
             {
+                const auto error = Napi::Error::New(env, result.error());
                 if (!m_onerrorHandlerRef.IsEmpty())
                 {
-                    m_onerrorHandlerRef.Call({});
+                    m_onerrorHandlerRef.Call({error.Value()});
+                    return;
                 }
 
-                throw Napi::Error::New(env, result.error());
+                throw error;
```

Comment:
Where does this error end up if it's thrown? Should this be the same as the other ones?

Reply:
I think you're looking at an old version of the PR?